lib/decode.js

```diff
@@ -55,12 +61,12 @@ function decode (data) {
         typetags = typetags.value;
         // so we start building our message list
         if (typetags[0] !== ',') {
-            throw 'invalid type tag in incoming OSC message, must start with comma';
+            throw new Error('invalid type tag in incoming OSC message, must start with comma');
```

Comment:
Decode is wrapped in a try/catch and the error is not passed to the user. Instead you log `e.message` which is undefined when you throw a String. I can move this to another pull request but the only thing this change does is log the error message instead of undefined.